e2etests/web/text_editing/.flutter-plugins-dependencies

```diff
@@ -0,0 +1 @@
+{"info":"This is a generated file; do not edit or check into version control.","plugins":{"ios":[{"name":"e2e","path":"/usr/local/google/home/nurhan/.pub-cache/hosted/pub.dartlang.org/e2e-0.2.4+4/","dependencies":[]}],"android":[{"name":"e2e","path":"/usr/local/google/home/nurhan/.pub-cache/hosted/pub.dartlang.org/e2e-0.2.4+4/","dependencies":[]}],"macos":[],"linux":[],"windows":[],"web":[]},"dependencyGraph":[{"name":"e2e","dependencies":[]}],"date_created":"2020-03-04 08:57:54.149356","version":"1.15.4-pre.257"}
```

Comment:
It says "do not edit or check into version control." 😄 

Let's add these auto-generated files to `.gitignore`.

Reply:
aha! thanks! :)

Reply:
done.